streaming/streaming.go

```diff
@@ -302,42 +306,73 @@ func LaunchStream(ctx context.Context, driver string, dbList []profiles.LogicalS
 }
 
 func CleanUpStreamingJobs(ctx context.Context, conv *internal.Conv, projectID, region string) error {
+	//create clients
 	c, err := dataflow.NewJobsV1Beta3Client(ctx)
 	if err != nil {
 		return fmt.Errorf("could not create job client: %v", err)
 	}
 	defer c.Close()
 	fmt.Println("Created dataflow job client...")
+	dsClient, err := datastream.NewClient(ctx)
+	fmt.Println("Created datastream client...")
+	if err != nil {
+		return fmt.Errorf("datastream client can not be created: %v", err)
+	}
+	defer dsClient.Close()
+
+	//clean up for single instance migrations
+	if conv.Audit.StreamingStats.DataflowJobId != "" {
```

Comment:
Yes, its possible, but this is the approach I have taken in all areas of the code where the code path for single instance is separate from sharding flow. This is to avoid introducing bugs in the single instance flow.
After sometime, we will deprecate the single instance flow altogether.